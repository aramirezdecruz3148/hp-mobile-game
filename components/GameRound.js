import React from "react";
import { View, Text } from "react-native";
import PropTypes from "prop-types";

export default function GameRound({ round }) {
  return (
    <View>
      <Text>Round: {round}</Text>
    </View>
  );
}

GameRound.propTypes = {
  round: PropTypes.number.isRequired
};
