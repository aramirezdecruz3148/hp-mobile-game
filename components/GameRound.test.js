import React from 'react';
import { shallow } from 'enzyme';
import GameRound from './GameRound';

describe('GameRound component', () => {
  it('renders GameRound', () => {
    const wrapper = shallow(<GameRound round={2}/>);
    expect(wrapper).toMatchSnapshot();
  });
});