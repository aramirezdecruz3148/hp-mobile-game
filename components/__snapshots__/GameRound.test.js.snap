// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameRound component renders GameRound 1`] = `
<View>
  <Text>
    Round: 
    2
  </Text>
</View>
`;
